lib/av_client.ts

```diff
@@ -264,6 +264,40 @@ export class AVClient implements IAVClient {
     return this.createVoterRegistration();
   }
 
+  public async expireVoterSessions(votingRoundReference: string): Promise<void> {
+    const {
+      url: vaUrl,
+      contextUuid: vaUuid,
+      authorizationMode,
+    } = this.getLatestConfig().items.voterAuthorizerConfig.content.voterAuthorizer;
+    const latestConfigAddress = this.getLatestConfig().items.latestConfigItem.address;
+    const coordinator = new VoterAuthorizationCoordinator(vaUrl, vaUuid);
+
+    let authorizationResponse: AxiosResponse
+
+    switch (authorizationMode) {
+      case "proof-of-election-codes":
+        authorizationResponse = await coordinator.authorizeProofOfElectionCodes(
+          this.keyPair.publicKey,
+          this.proofOfElectionCodes,
+          votingRoundReference,
+          "expire");
+        break;
+      case "proof-of-identity":
+        throw new InvalidStateError("voter_authorizer.expire_voter.proof_of_identity_not_supported");
+      default:
+        throw new InvalidConfigError(`Unknown authorization mode of voter authorizer: '${authorizationMode}'`);
+    }
+
+    const { authToken } = authorizationResponse.data;
+    const decodedAuthToken = jwtDecode<JwtPayload>(authToken);
+
+    if(decodedAuthToken === null)
+      throw new InvalidTokenError('Auth token could not be decoded');
+
+    await this.bulletinBoard.expireVoterSessions(authToken, latestConfigAddress);
```

Comment:
return the result of this response such that the application using this could for example log them out also?